app/app.go

```diff
@@ -257,6 +251,15 @@ func NewWasmApp(
 	supportedFeatures := "staking"
 	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.accountKeeper, app.bankKeeper, app.stakingKeeper, wasmRouter, wasmDir, wasmConfig, supportedFeatures, nil, nil)
 
+	if len(wasm.DefaultEnabledProposals) != 0 {
```

Comment:
Nit: maybe add a comment above this that people can define a list of enabled proposals in their `app.go` and not need to use the list from `x/wasm`

(As we encourage people to import `x/wasm` and use `app.go` as an integration reference)

Reply:
I would address this with https://github.com/CosmWasm/wasmd/issues/175 . Just added a note there